### PR TITLE
[DOCS] Python 3.7 support removal from docs for master

### PR DESCRIPTION
**Pull request by msmykx-intel**
Removing Python 3.7 from supported versions in documentation.

### CODE REVIEW

File: docs/dev/build_linux.md [hunk 1]
```diff
@@ -74,9 +74,9 @@ You can use the following additional build options:
      ```
   2. Enable the `-DENABLE_PYTHON=ON` option in the CMake step above (Step 4). To specify an exact Python version, use the following options:
      ```
-     -DPYTHON_EXECUTABLE=`which python3.7` \
-     -DPYTHON_LIBRARY=/usr/lib/x86_64-linux-gnu/libpython3.7m.so \
-     -DPYTHON_INCLUDE_DIR=/usr/include/python3.7
+     -DPYTHON_EXECUTABLE=`which python3.8` \
+     -DPYTHON_LIBRARY=/usr/lib/x86_64-linux-gnu/libpython3.8m.so \
```
Comment (akuporos) [inline docs/dev/build_linux.md:78]: as far as I know there is no `libpython3.8m.so` in this dir

CC @ilya-lavrenov @jiwaszki

Comment (ilya-lavrenov) [reply docs/dev/build_linux.md:78]: `m` is obsolete and used in 3.7 and older

File: docs/dev/build_raspbian.md [hunk 2]
```diff
@@ -43,9 +43,9 @@ git clone --recurse-submodules --single-branch --branch=master https://github.co
   via `pip3`, adding the following options:
    ```sh
    -DENABLE_PYTHON=ON \
-   -DPYTHON_EXECUTABLE=/usr/bin/python3.7 \
-   -DPYTHON_LIBRARY=/usr/lib/arm-linux-gnueabihf/libpython3.7m.so \
-   -DPYTHON_INCLUDE_DIR=/usr/include/python3.7
+   -DPYTHON_EXECUTABLE=/usr/bin/python3.8 \
+   -DPYTHON_LIBRARY=/usr/lib/arm-linux-gnueabihf/libpython3.8m.so \
```
Comment (ilya-lavrenov) [inline docs/dev/build_raspbian.md:47]: remove `m` suffix in other place in docs as well

**Review by ilya-lavrenov — CHANGES_REQUESTED** [review]

File: docs/dev/build_raspbian.md (hunk 2 above)
Comment (msmykx-intel) [reply docs/dev/build_raspbian.md]: Done in https://github.com/openvinotoolkit/openvino/pull/19670/commits/eb4ec4338a562277b675eb6a208aa723f07b3b6a

**Review by ilya-lavrenov — APPROVED** [review]

**Review by akuporos — APPROVED** [review]

**Pull request merged**